cli/internal/run/run.go

```diff
@@ -941,10 +906,7 @@ func (r *run) generateDotGraph(taskGraph *dag.AcyclicGraph, outputFilename fs.Ab
 			r.ui.Output(fmt.Sprintf("✔ Generated task graph in %s", ui.Bold(outputFilename.ToString())))
 		}
 	} else {
-		r.ui.Output("")
-		r.ui.Warn(color.New(color.FgYellow, color.Bold, color.ReverseVideo).Sprint(" WARNING ") + color.YellowString(" `turbo` uses Graphviz to generate an image of your\ngraph, but Graphviz isn't installed on this machine.\n\nYou can download Graphviz from https://graphviz.org/download.\n\nIn the meantime, you can use this string output with an\nonline Dot graph viewer."))
-		r.ui.Output("")
-		r.ui.Output(graphString)
+		r.generateDotGraph(graphString, true)
```

Comment:
What is `true` here? Do we ever pass `false`? If we do in fact need this boolean, I would consider assigning it to an appropriately named local variable so that its meaning is clear.

Reply:
Yea I agree this needs to be made more clear. 

This is a flag used to show or hide the warning about installing graphviz. We used to always show this warning when displaying the dotGraph. But now that the dotGraph can be displayed _intentionally_, I swapped this to only display when you asked to generate a graph to a file, but didn't have graphviz installed (so we are forced to fallback to dotGraph). 

If you just ask for the dotGraph, you won't see the warning about installing graphviz.